frost-core/src/frost/round1.rs

```diff
@@ -31,12 +31,21 @@ where
     where
         R: CryptoRng + RngCore,
     {
-        let mut k_enc = [0; 32];
-        rng.fill_bytes(&mut k_enc[..]);
+        let mut random_bytes = [0; 32];
+        rng.fill_bytes(&mut random_bytes[..]);
 
+        Self::nonce_generate_from_random_bytes(secret, random_bytes)
+    }
+
+    /// Generates a nonce from the given random bytes.
+    /// This function allows testing and MUST NOT be made public.
```

Comment:
I'm assuming this is strongly enforced in configuration somewhere?

Reply:
This is just a reminder to developers (i.e. ourselves in the future :laughing: ) to not make the function `pub`